pimcore/lib/Pimcore/Console/Command/ThumbnailsImageCommand.php

```diff
@@ -98,7 +98,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 }
 
                 if ($input->getOption("system")) {
-                    $thumbnail = Asset\Image\Thumbnail\Config::getPreviewConfig();
+                    $thumbnail = \Pimcore::getDiContainer()->get('Pimcore\Model\Asset\Image\Thumbnail\Config');
+                    $thumbnail = $thumbnail::getPreviewConfig();
```

Comment:
Yes, you are right, but the DI Container will create it only once so it's a singleton.

So, I thought about a better solution and that would be a factory. Since ::getPreviewConfig() is already a configuration factory, I would suggest doing this:
```php
	'Pimcore\Model\Asset\Image\Thumbnail\Config::getPreviewConfig' => DI\factory('Pimcore\Model\Asset\Image\Thumbnail\Config::getPreviewConfig'),
	'Pimcore\Model\Asset\Video\Thumbnail\Config::getPreviewConfig' => DI\factory('Pimcore\Model\Asset\Video\Thumbnail\Config::getPreviewConfig'),
`
Now it would be only matter to define what to use as key. "Pimcore\Model\Asset\Video\Thumbnail\Config::getPreviewConfig" is rather long, but makes sense to use. What would you suggest?

Thanks